managed/services/backup/compatibility_service_test.go

```diff
@@ -131,16 +151,61 @@ func TestCheckCompatibility(t *testing.T) {
 			},
 			expectedError: nil,
 		},
+		// mongodb cases
+		{
+			name:        "successful",
+			serviceType: models.MongoDBServiceType,
+			versions: []agents.Version{
+				{Version: "6.0.2-1"},
```

Comment:
As we agreed with @rnovikovP for now we just limit the minimum version of pbm. Any other constraints can be added later when we have clear understanding of the topic. Since we don't have any agreement with pbm team about versions supporting, just using some value from external tool docs makes no sense.